redi-connect-front/src/pages/app/profile/Profile.tsx

```diff
@@ -45,14 +49,45 @@ function Profile ({ loading, profile, currentUser, profilesFetchOneStart }: Prof
     <>
       <FullScreenCircle loading={loading} />
       {!loading && <LoggedIn>
-        {(!isAcceptedMatch || currentUserIsMentor) && (
-          <Button
-            onClick={() => history.goBack()}
-            simple
-          >
-            Back
-          </Button>
-        )}
+
+        <Columns>
+          <Columns.Column>
+            {(!isAcceptedMatch || currentUserIsMentor) &&
```

Comment:
There are multiple heavy boolean expressions for conditional rendering in this single return statement. Let's hold off sophisticated breaking up for now, but let's pick the low-hanging high-value fruit of moving these boolean expressions to right above the return statement, and give them descriptive names.

For this one I suggest:
`const isCurrentUserAMentorReviewingAPendingApplication = !isAcceptedMatch || currentUserIsMentor`

Reply:
I will leave the comment for the profile out as the logic is still the old. The page will be refactored anyway.